examples/erc20-flashmint/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+#![cfg_attr(not(test), no_main)]
```

Comment:
Flash Mint tests are already implemented in a [separate PR](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/407), remove them from this PR.

Reply:
+1